stable/spring-cloud-data-flow/values.yaml

```diff
@@ -37,11 +37,17 @@ server:
   #    cpu: 0.5
   #    memory: 640Mi
   extraEnv: {}
+  containerConfiguration: {}
```

Comment:
Don't we want to highlight this in the README as an option? Perhaps point to the docs in the Microsite/ref.guide?

Reply:
README updated for both props